spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/config/SkipperServerConfiguration.java

```diff
@@ -183,7 +185,7 @@ public ReleaseService releaseService(PackageMetadataRepository packageMetadataRe
 	}
 
 	@Bean
-	@ConditionalOnProperty("spring.cloud.skipper.server.disableReleaseStateUpdateService")
+	@ConditionalOnProperty(value = "spring.cloud.skipper.server.enableReleaseStateUpdateService", matchIfMissing = true)
```

Comment:
We could use `prefix`, something like: `@ConditionalOnProperty(prefix = "spring.cloud.skipper.server", name= "enableReleaseStateUpdateService",
			matchIfMissing = true)`